src/Database/Connection.php

```diff
@@ -158,10 +159,11 @@ public function driver($driver = null, $config = [])
             return $this->_driver;
         }
         if (is_string($driver)) {
-            if (!class_exists($driver)) {
+            $className = App::className($driver, 'Database/Driver');
+            if (!$className || ($className && !class_exists($className))) {
```

Comment:
The `$className && ` is not necessary as `||` bails early if the first condition is not true.